console/input.rst

```diff
@@ -311,6 +311,37 @@ The above code can be simplified as follows because ``false !== null``::
     $yell = ($optionValue !== false);
     $yellLouder = ($optionValue === 'louder');
 
+Fetching The Raw Command Input
+------------------------------
+
+Sometimes, you may need to fetch the raw input that was passed to the command.
+This is useful when you need to parse the input yourself or when you need to
+pass the input to another command without having to worry about the number
+of arguments or options defined in your own command. This can be achieved
+thanks to the
+:method:`Symfony\\Component\\Console\\Input\\ArgvInput::getRawTokens` method::
+
+    // ...
+    use Symfony\Component\Process\Process;
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        // pass the raw input of your command to the "ls" command
+        $process = new Process(['ls', ...$input->getRawTokens(true)]);
```

Comment:
I still have problems understanding what "a token" is in this context.

Imagine that I run a command like this:

```
$ php bin/console app:my-command -c -a --foo --bar=zzz arg1 -- arg2 -b
```

Is this the content of `getRawTokens()`?

```
['-c', '-a', '--foo', '--bar=zzz', 'arg1', '--', 'arg2', '-b']
```

Also, what's the purpose of the boolean argument of `getRawTokens()`?

Thanks!

Reply:
I guess it is called a token because it can be anything: letters, pipes, dashes, etc. Not sure but that what makes the more sense to me 🙂 

About the boolean argument, I wrote a sentence about it just under the code snippet. Should this also be present in the code snippet?

Reply:
I will be back in 2 days, and I have an idea in mind. Will add a proposal then.

I am currently on a phone 📱

Reply:
At the end I merged this after rewording it a bit and expanding the code examples. If we don't like anything about this, let's create a new issue or PR. Thanks!